agent/config/config_test.go

```diff
@@ -214,6 +217,28 @@ func TestInvalidDockerStopTimeout(t *testing.T) {
 	assert.Error(t, conf.validateAndOverrideBounds(), "Should be error with negative DockerStopTimeout")
 }
 
+func TestInvalidFormatContainerStartTimeout(t *testing.T) {
+	defer setTestRegion()()
+	defer setTestEnv("ECS_CONTAINER_START_TIMEOUT", "invalid")()
+	conf, err := environmentConfig()
+	assert.NoError(t, err)
+	assert.Zero(t, conf.ContainerStartTimeout, "Wrong value for ContainerStartTimeout")
+}
+
+func TestInvalidValueContainerStartTimeout(t *testing.T) {
+	defer setTestRegion()()
+	defer setTestEnv("ECS_CONTAINER_START_TIMEOUT", "-10s")()
+	conf, err := environmentConfig()
+	assert.NoError(t, err)
+	assert.Zero(t, conf.ContainerStartTimeout)
+}
+
+func TestInvalidContainerStartTimeout(t *testing.T) {
```

Comment:
I don't think we actually need this test as it does not add much value. Creating a config object should always be from the DefaultConfig() or envConfig() or other methods similar to those.

Reply:
OK, this test will be removed.